### ISSUE TITLE
Board Philosophy 

### ISSUE THREAD

**Issue by sweetsbeats** [barrucadu/gyuunu#17]
My idea for gyuunu originally, was spawned out of the particularly distasteful posting seen on well known western boards like 4chan, eventually 8chan, meguca, ect. Users focused less on the discussion, and more on trying to "BTFO" and call out users for subjective tastes and interests using cringy greentexting and smug anime pictures(its funny in jest, but "just pretending to be retarded" for 30+ posts isn't). After acclimating to boards like /tea/ and sushi, I got the idea to have a board where users are asked to keep it classy, which gave way to the original attempt(the meguca forking failure).  My ideal board is shaped around preventing some of the bad experiences from imageboards, while trying to maintain some integrity, and not devolving into a tripcode cesspit. 

To describe the end goal I had originally;

-A cool front page like sushi's, but with info on board by board post count, frequency, ect. (like 8chan's)
 
-4 boards; anime & manga, Video games, tech, and random. (Random would be a test board, where every 1-2 week(s) there's a new feature/gimmick to encourage users to come and post. At the end of the period, posts would be wiped. *

-Naming Scheme revolving around milk (for example, the games board would be /arcade/, and would be named "Strawberry Milk Arcade", or that in Japanese)

-Users encouraged to make generals for topics relative to more niche boards( for example, for /tg/ related posting, users are encouraged to make a /tg/general **

- Users are ALL anonymous; if one needs a trip it is requested along with a fair reason. (let's say you're a game dev answering questions, you would need a trip. If you're trying to use your /bun/ trip and it isnt working, thats not a fair reason.)

-Lewds should be spoilered in non-dedicated threads; dedicated threads must be clearly labeled and opening images be spoilered

- Extreme types of lewds (furry, gore, scat, vore, ect. ) are explicitly forbidden site-wide

 *I realize now that the random board is most likely not feesable, as well as my hopes for how the community will act.

  **The thought process behind this idea is; humans have a natural tendency to divide, if they can, they will.
8chan was a perfect example of how communities fracture into sub-communities, and lower the frequency of posts(I can elaborate on this more if you would like to know). Lower frequency of posts means its a slower board, and slower board means less people come, meaning the community looks smaller than it might be. Multiple boards also means multiple tabs open for 2+ different catalogs, which is more effort. Having one overarching board that covers the general topic will encourage users to peruse other threads that they might not have seen had they not gone to that specific board, and may post more.

To summarize the philosophy;
1.) Anonymity is KEY (tripcodes are a privilege; they're for people who SHOULD be identified, not WANT to)
2.) Make threads, not boards
3.) Discourage "call out culture"(terrible name for it.) and encourage encourage good posting
4.) Be cool!

**Comment by barrucadu** [comment]
> anime & manga, Video games, tech

Something that comes to mind is that these are incredibly generic. Go pretty much anywhere other than /tea/, and you'll find boards for these topics. So why would someone come to a much smaller community when there are already so many established ones?
